setup.py

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/python
```

Comment:
On July 9, 2015 6:53:11 PM EDT, Lee Clemens <notifications@github.com> wrote:
>> @@ -1,4 +1,3 @@
>> -#!/usr/bin/python
>
>I can't recall any documentation for a Python project saying to run
>`./setup.py install`, 

Because wouldn't work on Windows :-) our another system where shebang wouldn't point to installed python


> but I already put the shebang back in :)

 Yeay!